Dockerfile

```diff
@@ -99,19 +98,8 @@ RUN NODE_ENV='production' yarn build
 ####################################################################################################
 FROM golang:1.14.1 as argocd-build
 
-COPY --from=builder /usr/local/bin/dep /usr/local/bin/dep
 COPY --from=builder /usr/local/bin/packr /usr/local/bin/packr
 
-# A dummy directory is created under $GOPATH/src/dummy so we are able to use dep
-# to install all the packages of our dep lock file
-COPY Gopkg.toml ${GOPATH}/src/dummy/Gopkg.toml
```

Comment:
Can you please add following to cache dependencies:

```
COPY go.mod /src/go.mod
COPY go.sum /src/go.sum

RUN go mod download
```

Reply:
Done, please have another look